server/lorax_server/models/model.py

```diff
@@ -261,9 +261,9 @@ def load_batched_adapter_weights(
         lora_a_list = [pad_rank(w, dim=1, world_size=self.world_size) for w in lora_a_list]
         lora_b_list = [pad_rank(w, dim=0, world_size=self.world_size) for w in lora_b_list]
 
-        if lora_b_list:
+        if lora_a_list:
```

Comment:
Curious what the significance of swapping from `lora_b_list` to `lora_a_list` here is.

Reply:
It is the same, actually.